readme.md

```diff
@@ -226,6 +226,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Tutorials](https://github.com/ujjwalkarn/Machine-Learning-Tutorials)
 - [Speech and Natural Language Processing](https://github.com/edobashira/speech-language-processing)
 	- [Spanish](https://github.com/dav009/awesome-spanish-nlp)
+	- [NLP with Ruby](https://github.com/arbox/Awesome-Natural-Language-Processing-with-Ruby) - Ruby focused NLP tools and resources.
```

Comment:
`tools and resources` are moot. Almost all lists have these. I don't think it needs a description, as it's just a repetition of the name anyways.